src/main/resources/db/migration/V189__Restructure_Instructor_Costs.sql

```diff
@@ -0,0 +1,17 @@
+ALTER TABLE `InstructorTypes` RENAME TO `InstructorTypeCosts`;
+ALTER TABLE `InstructorTypeCosts` ADD COLUMN `InstructorTypeId` int(11) NULL;
+
+ALTER TABLE `InstructorCosts` CHANGE COLUMN `InstructorTypeId` `InstructorTypeCostId` int(11) NULL;
+
+CREATE TABLE `InstructorTypes` (
+  `Id` INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  `Description` varchar(50) NOT NULL
```

Comment:
32 should be long enough. Excessive length can slow down the database presumably.